static/js/components/GroupSources.jsx

```diff
@@ -238,6 +259,11 @@ const GroupSources = ({ route }) => {
     );
   };
 
+  const renderDateSaved = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_date_saved`}>{source.created_at}</div>;
```

Comment:
Use `source.saved_at`

Reply:
That does not exist. As far as I can tell, for sources "created_at" represents when the object was saved.

Reply:
https://github.com/skyportal/skyportal/blob/9c986c12407b9cf418088d7a2b29965b140fc580/skyportal/models.py#L787